IssueTracker/DataAccess/Data/IData/IParticipantData.cs

```diff
@@ -9,6 +9,6 @@ public interface IParticipantData
         Task<IEnumerable<Participant>> GetAllAsync();
         Task<Participant?> GetByIdAsync(int id);
         Task UpdateAsync(Participant participant);
-        Task<IEnumerable<Participant>> GetByProjectIdAsync(string storedProcedure, int id, string connectionId = "Default");
+        Task<IEnumerable<Participant>> GetByProjectIdAsync(int id, string storedProcedure = "dbo.spParticipant_GetAllByProjectId", string connectionId = "Default");
```

Comment:
DB details, like stored procedures (or connections strings) should not be exposed here

Reply:
Task<IEnumerable<Participant>> GetByProjectIdAsync(int id)